README.md

```diff
@@ -60,3 +60,32 @@ We collect messages to provide platform without abuse, advertisement.
 - information that break copyright law (or any of it analogue) ©
 - advertisement
 - and staff that can violate law
+
+## Scenario of usage
```

Comment:
I can't return to previous chat?

Reply:
No, you can not. The idea is anonymity and random chat.